Dockerfile.dev

```diff
@@ -0,0 +1,12 @@
+FROM docker.io/node:16
```

Comment:
Docker repositories are bit restrictive at times, would it be possible to use `registry.access.redhat.com/ubi8/nodejs-16`  for this?

I've tried to figure out what https://github.com/RedHatInsights/insights-frontend-builder-common/blob/be2fa6ca7dd7a84f8efce9cdd93febf8e541a5ce/src/frontend-build.sh#L145 does, but I failed to run it localy :/

Reply:
I also think we should stick with Red Hat certified containers. All of our dependencies should work as they also use them.